docker-compose.yaml

```diff
@@ -51,8 +51,8 @@ services:
       options:
         max-size: "20m"
     ports:
-      - "127.0.0.1:9944:9944"
-      - "127.0.0.1:9933:9933"
+      - "9944:9944"
```

Comment:
The problem here is that you've opened access to ports `9944` and `9933` not only from localhost and it is a security issue that previously has been reported in the audit. 
To avoid it, I suggest getting/setting the required data from/to nodes by using an ssh proxy always. 
For instance, for the port `9944` it will be `ssh -L 9944:localhost:9944 -tt user@ip`.